spec/requests/blocked_ip_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
This spec is in a new directory `spec/requests` which didn't exist before. We should add that to the [Github actions matrix](https://github.com/inaturalist/inaturalist/blob/main/.github/workflows/CI-build-test.yml) to make sure it gets run during CI - probably in the last entry with helpers, initializers, etc